website/docs/tutorials/04-configuration-file.md

```diff
@@ -35,65 +38,98 @@ network = "testnet"
 ```
 
-:::note
-Currently, [`protostar build`](/docs/cli-reference#build) requires this section.
-:::
-
 ### `[COMMAND]`
-Configure command arguments, to avoid passing them manually. CLI can be represented in the configuration file in the following way:
+The `[COMMAND]` section of the `protostar.toml` file allows you to specify options and settings for a specific Protostar command.
 
-```console title="CLI"
-protostar format src tests --ignore-broken
+To configure command arguments in the `protostar.toml` file, you can specify them in a `[COMMAND]` section using the following format:
+
+```toml
+[COMMAND]
+flag = true
+list = [1, 2, 3]
+path = "./src"
 ```
+
+For example, the following configuration file specifies the [`target`](/docs/cli-reference#target-string) and [`ignore-broken`](/docs/cli-reference#--ignore-broken) arguments for the [`protostar format` command](/docs/cli-reference#format):
+
 ```toml title="Configuration File"
 [format]
 target = ["src", "tests"]
 ignore-broken = true
 ```
 
-Check the [CLI Reference](/docs/cli-reference) or run `protostar COMMAND --help` to learn supported arguments.
+You can then run the [`protostar format`](/docs/cli-reference#format) command without specifying the [`target`](/docs/cli-reference#target-string) and [`ignore-broken`](/docs/cli-reference#--ignore-broken) arguments in the console:
+
+```console title="CLI"
+protostar format
+```
+
+To learn more about the available options and arguments for each Protostar command, refer to the [CLI Reference](/docs/cli-reference) or run `protostar COMMAND --help` to see the list of supported arguments.
 
 ### Configuration Profiles
-Configuration profiles provide a way to easily switch between Protostar configurations. Profiles inherit values from non-profiled configuration. In order to create a configuration profile, add a new section in protostar.toml with the following naming convention:
--  `[profile.PROFILE_NAME.project]` - to create a profile for a [Shared Configuration](#shared-configuration)
--  `[profile.PROFILE_NAME.COMMAND]` - to create a profile for a [Command Configuration](#command)
+Configuration profiles allow you to easily switch between different Protostar configurations. 
+When you activate a profile, it will override the default settings specified in the `protostar.toml` file with the settings specified in the profile.
 
-To activate the profile, use the profile argument
-```console title="Run the test command with the 'integration' profile"
-protostar -p integration test
-```
+To create a configuration profile, add a new section to the `protostar.toml` file with the following naming convention:
+-  `[profile.PROFILE_NAME.project]` - to create a [Shared Configuration](#shared-configuration) profile
+-  `[profile.PROFILE_NAME.COMMAND]` - to create a [Command Configuration](#command) profile
+
+To activate a profile, use the [-p or --profile argument](/docs/cli-reference#-p---profile-string) followed by the name of the profile:
```

Comment:
```suggestion
To use a profile, use the [-p or --profile argument](/docs/cli-reference#-p---profile-string) followed by the name of the profile:
```

Reply:
[replace activate with use](https://github.com/software-mansion/protostar/pull/1308/commits/717d490873eef2312bd6b1ac259ef91cca216104)

I didn't commit the suggestion, because "use" is used again after the comma.

Reply:
To use a profile, add <> flag 

maybe?

Reply:
The word activate is weird, use makes it clear and unambiguous

Reply:
I replaced the second "use" with "specify".

Reply:
[use add instead of specify](https://github.com/software-mansion/protostar/pull/1308/commits/5250e419beb272f9f160b253c6519bba1412e5e4)